soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6033,7 +6033,7 @@ void func_8083DFE0(Player* this, f32* arg1, s16* arg2) {
             }
         }
 
-        if (CVar_GetS32("gMMBunnyHood", 0) && this->currentMask == PLAYER_MASK_BUNNY) {
+        if (CVar_GetS32("gMMBunnyHood", 0) == 1 && this->currentMask == PLAYER_MASK_BUNNY) {
```

Comment:
Could it be this is missing something? This only makes it so the old setting is moved to the combobox (fast + jump velocity), but I'm not seeing anything to accomodate for only ground speed.

Reply:
Ah, never mind, the ground speed and jump speed were already 2 seperate things, this just makes it so one of them doesn't apply depending on settings. I wasn't aware of this.